rasa/core/policies/mapping_policy.py

```diff
@@ -91,7 +92,8 @@ def predict_action_probabilities(
         predicted with the highest probability of all policies. If it is not
         the policy will predict zero for every action."""
 
-        prediction = self._default_predictions(domain)
+        result = self._default_predictions(domain)
```

Comment:
I think `result` isn't very intuitive. How about something like: 
```suggestion
        action_confidences = self._default_predictions(domain)
```

Reply:
we use result in every policy, that's why I renamed it there as well

Reply:
I'd stick to `result`, it's shorter

Reply:
I don't agree. The length should be a criteria because we are all using auto completion in our IDEs. And `result` doesn't communicate anything, while something like `action_confidences` or `action_predictions` exactly state what's in there.

Reply:
long names make it also harder to understand not only type. but regardless renaming it everywhere is out of scope of this PR. Since it is everywhere for now, I'd keep result in mapping policy as well.

Reply:
I don't agree regarding the naming, but it shouldn't block this